boards/xtensa/intel_s1000_crb/intel_s1000_crb.dts

```diff
@@ -35,11 +36,54 @@
 
 &spi0 {
 	status = "ok";
+	#address-cells = <1>;
+	#size-cells = <0>;
```

Comment:
You have #size-cells twice, I think you can probably remove both of them.

Reply:
I removed one #size-cells. Couldn't figure out which was the other one.